astropy/visualization/wcsaxes/tests/test_frame.py

```diff
@@ -37,8 +36,7 @@ def update_spines(self):
 class TestFrame(BaseImageTests):
 
     @pytest.mark.remote_data(source='astropy')
-    @pytest.mark.mpl_image_compare(baseline_dir=IMAGE_REFERENCE_DIR,
-                                   tolerance=0, style={})
+    @pytest.mark.mpl_image_compare(tolerance=0, style={})
```

Comment:
Would it not be worth making an `astropy_image_compare` decorator which wraps all these arguments? Also you could arguably want to increase the tolerance from 0 at this point, because the hashes are doing that and the tolerance number is now interpreted as "below this line the hash is wrong, just update it".

Reply:
I would prefer if we could somehow set these options globally in pytest-mpl without having to have a custom decorator. First attempt here: https://github.com/matplotlib/pytest-mpl but needs some work. Could we leave this as-is for now and try and simplify later if we can improve pytest-mpl?